guibot/finder.py

```diff
@@ -255,6 +256,7 @@ def from_match_file(filename):
                         param = param_string
                     finder.params[category][option] = param
 
+        finder.synchronize()
```

Comment:
Interesting, just wondering for the sake of commenting here: what case did you encounter to unveil this?

Reply:
Also, did the unit tests you suggested in #13 help you for this?

Reply:
> Interesting, just wondering for the sake of commenting here: what case did you encounter to unveil this?

IIRC it happened when loading a `.match` file generated from a `Text` finder. Since the synchronization happened before the parameters were read, it was using default values and trying to load the classifier [here](https://github.com/intra2net/guibot/blob/master/guibot/finder.py#L1951).

> Also, did the unit tests you suggested in #13 help you for this?

Yes, they introduce unit tests for `Chain` targets, covering the scenario when text targets are loaded from match files.